package.json

```diff
@@ -7,12 +7,12 @@
     "ros"
   ],
   "scripts": {
-    "test": "mocha test/directory.js",
+    "test": "mocha --exit test/directory.js",
     "gennodejsTest": "mocha test/gennodejsTest.js test/onTheFlyMessages.js",
     "stressTest": "mocha test/stress.js",
     "flatten": "node tools/flatten.js",
     "generate": "node dist/tools/generateMessages.js",
-    "compile": "babel --presets es2015 src/ -d dist/",
+    "compile": "babel --presets env src/ -d dist/",
```

Comment:
I think we'd need to add targets here too, right? The compilation is meant to happen before publishing the package to npm so that the published code is compatible with older versions of Node. My read of the doc for babel-preset-env is that this will just transpile for the current version.

Reply:
For sure, which target would you like the package to support? Can definitely add the flag, or just add a babelrc file for this